### PR TITLE
React 508 continued

### PR DESCRIPTION
**Pull request by sarnikbixal**
Thanks for submitting a pull request! Below are a few things you can do to help us more quickly review your changes.

## Checklist

I have…

- [x] run the application locally (`make up`) and verified that my changes behave as expected.
- [ ] run static behat test suite (`circleci build --job behat`) against my changes.
- [x] run the code sniffer (`circleci build --job code-sniffer`) against my changes.
- [x] run the code coverage tool (`circleci build --job code-coverage`) 
      against my changes
- [ ] summarized below my changes and noted which issues (if any) this pull request fixes or addresses.
- [ ] thoroughly outlined below the steps necessary to test my changes.
- [ ] attached screenshots illustrating relevant behavior before and after my changes.
- [ ] read, understand, and agree to the terms described in [CONTRIBUTING.md](https://github.com/Bixal/move.mil/blob/master/CONTRIBUTING.md).
- [ ] added my name, email address, and copyright date to [CONTRIBUTORS.md](https://github.com/Bixal/move.mil/blob/master/CONTRIBUTORS.md).

## Summary of Changes

This pull request…

- _Summarize change number one._
- _Summarize change number two._
- _etc. etc. etc._

## Testing

To verify the changes proposed in this pull request…

1. _Describe testing step one._
1. _Describe testing step two._
1. _etc. etc. etc._

## Screenshots

_Attach relevant before and after screenshots here._

### CODE REVIEW

File: web/modules/custom/react_tools/tools/apps/react-ppm-tool/src/components/weights.js [hunk 1]
```diff
@@ -46,24 +47,26 @@ class Weights extends Component {
                     </div>
                 </div>
                 <div className="weight-item">
-                    <div>Estimated Pro-Gear Weight (lbs)</div>
-                        <Input validationType="number" 
-                               value={this.props.weightOptions.proGear}
-                               type="text" 
-                               onChangeFn={this.handleChange} 
-                               params={{key: 'proGear'}} />
+                    <Input  id="proGeart"
```
Comment (ivonne-gov) [inline web/modules/custom/react_tools/tools/apps/react-ppm-tool/src/components/weights.js]: proGear instead of proGeart

**Comment by ivonne-gov** [comment]
hey, @sarnikbixal I cannot enter numbers (or text actually) in this input text, maybe it's because of the id of the previous comment? (PPM tool)

screen shot 2018-08-23 at 10 23 23 am

**Comment by ivonne-gov** [comment]
the validation on pro gear field was removed, IIRC it was there, so maybe some of the new labels are messing with the validation?
screen shot 2018-08-23 at 10 28 04 am

**Comment by sarnikbixal** [comment]
@ivonne-gov I am not sure why you can not put numbers into the input, I tested it and mine works fine, as for the validation, it is only required on the FIRST field. This was how the old site worked as well.

**Comment by ivonne-gov** [comment]
@sarnikbixal you're right about the validation, I checked on staging and it doesn't have it in there. I thought it was an error of this PR. So, you can ignore that comment.

**Pull request merged**